From ba725a3d724357d8d68d21df8cdc1a0ffe5a908e Mon Sep 17 00:00:00 2001
From: johnmart19 <johnivan19999@gmail.com>
Date: Sun, 15 Jan 2023 00:11:11 +0200
Subject: [PATCH] Piex: Import Xiaomi camera hal changes

- Fixing issue with missing symbols " _ZN4piex19GetPreviewImageDataEPNS_15StreamInterfaceEPNS_16PreviewImageDataE "

Change-Id: I05f964b0f0f50afe1466eb8ffdde973d86cc1b1f
---
 src/piex.cc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/piex.cc b/src/piex.cc
index ac2ef0b..c800266 100644
--- a/src/piex.cc
+++ b/src/piex.cc
@@ -692,6 +692,11 @@ Error GetPreviewImageData(StreamInterface* data,
       return kUnsupported;
   }
 }
+Error GetPreviewImageData(StreamInterface* data,
+                          PreviewImageData* preview_image_data)
+{
+  return(GetPreviewImageData(data,preview_image_data,nullptr));
+}
 
 bool GetDngInformation(StreamInterface* data, std::uint32_t* width,
                        std::uint32_t* height,
-- 
2.34.1

